index.js

```diff
@@ -88,7 +89,8 @@ function processStream(options) {
         if (options.outputFile) {
             fs.writeFileSync(options.outputFile, serializedResult, 'utf-8');
         } else {
-            console.log(serializedResult);
+            const result = options.noColor ? serializeResult : colorize(serializedResult);
```

Comment:
should be `options.color` – clap removes `no-` and inverts value

Reply:
fixed